pdc_client/__init__.py

```diff
@@ -97,7 +98,7 @@ class PDCClient(object):
     connections. The authentication token is automatically retrieved (if
     needed).
     """
-    def __init__(self, server, token=None, develop=None, ssl_verify=None, page_size=None):
+    def __init__(self, server, token=None, develop=None, ssl_verify=None, page_size=None, page=None):
```

Comment:
OK, now I understand.

But we shouldn't add private/internal stuff to public API. `PDCClient.__init__()` is part of public API.

Having `page_size` there makes sense since it can be used for all request. But it's only used in `get_paged()` and not in `__getitem__()` (I didn't notice this before). This looks like a bug to me.

Even though the parameter is not mentioned in docstring, which it should be, user can make quick assumptions about it based on its name. We should make it harder for the API to be misused.

For now, it would be better to set the parameter to `PDCClient` using some private interface. But I'm not sure what's the best way to solve this in Python.